src/SentrySdk.php

```diff
@@ -0,0 +1,312 @@
+     *
+     * @param string   $message The message
+     * @param Severity $level   The severity level of the message
+     *
+     * @return string|null
+     */
+    public static function captureMessage(string $message, ?Severity $level = null): ?string
+    {
+        if (null === self::$currentHub) {
+            self::$currentHub = new Hub();
+        }
+
+        return self::$currentHub->captureMessage($message, $level);
+    }
+
+    /**
+     * Captures an exception event and sends it to Sentry.
+     *
+     * @param \Throwable $exception The exception
+     *
+     * @return string|null
+     */
+    public static function captureException(\Throwable $exception): ?string
+    {
+        if (null === self::$currentHub) {
+            self::$currentHub = new Hub();
+        }
+
+        return self::$currentHub->captureException($exception);
+    }
+
+    /**
+     * Captures a new event using the provided data.
+     *
+     * @param array $payload The data of the event being captured
+     *
+     * @return string|null
+     */
+    public static function captureEvent(array $payload): ?string
+    {
+        if (null === self::$currentHub) {
+            self::$currentHub = new Hub();
+        }
+
+        return self::$currentHub->captureEvent($payload);
+    }
+
+    /**
+     * Captures an event that logs the last occurred error.
+     *
+     * @return string|null
+     */
+    public static function captureLastError(): ?string
+    {
+        if (null === self::$currentHub) {
+            self::$currentHub = new Hub();
+        }
+
+        return self::$currentHub->captureLastError();
+    }
+
+    /**
+     * Records a new breadcrumb which will be attached to future events. They
+     * will be added to subsequent events to provide more context on user's
+     * actions prior to an error or crash.
+     *
+     * @param Breadcrumb $breadcrumb The breadcrumb to record
+     *
+     * @return bool Whether the breadcrumb was actually added to the current scope
+     */
+    public static function addBreadcrumb(Breadcrumb $breadcrumb): bool
+    {
+        if (null === self::$currentHub) {
+            self::$currentHub = new Hub();
+        }
+
+        return self::$currentHub->addBreadcrumb($breadcrumb);
+    }
+
+    /**
+     * Gets the integration whose FQCN matches the given one if it's available on the current client.
+     *
+     * @param string $className The FQCN of the integration
+     *
+     * @return IntegrationInterface|null
+     */
+    public static function getIntegration(string $className): ?IntegrationInterface
+    {
+        if (null === self::$currentHub) {
+            self::$currentHub = new Hub();
+        }
+
+        return self::$currentHub->getIntegration($className);
+    }
+
+    /**
+     * @see https://www.php.net/manual/en/language.oop5.cloning.php#object.clone
+     */
+    public function __clone()
+    {
+        throw new \BadMethodCallException('Cloning is forbidden.');
```

Comment:
If you are forbidding something it would be nice to have a tiny explanation why (you know, I know but someone reading this maybe doesn't). There are some other places too.

Reply:
I don't think that it adds value to specify that cloning a singleton is not allowed since ensuring that just one instance of the class exists during the execution of the program is the foundation of the concept of a singleton...

Reply:
I dropped the `__clone` and `__unserialize` methods as this class only have `static` methods and the constructor is `private`, thus there is no way to get an instance of this class (you could use reflection, but there is no reason to use it)